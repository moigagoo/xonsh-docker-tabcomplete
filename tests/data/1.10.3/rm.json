{
        "arguments": [
                "-f", "--force",
                "--help",
                "-l", "--link",
                "-v", "--volumes"
        ],
        "commands": []
}
