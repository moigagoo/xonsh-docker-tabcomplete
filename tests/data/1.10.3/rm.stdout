
Usage:	docker rm [OPTIONS] CONTAINER [CONTAINER...]

Remove one or more containers

  -f, --force        Force the removal of a running container (uses SIGKILL)
  --help             Print usage
  -l, --link         Remove the specified link
  -v, --volumes      Remove the volumes associated with the container
